apps/examples/performance/tls_handshake/README.txt

```diff
@@ -0,0 +1,11 @@
+examples/tls_handshake
+^^^^^^^^^^^^^^^^^^^^^
+  usage:
+    ex) tls_handshake
```

Comment:
Please update usages for TLS_HANDSHAKE_USAGE.
```
#define TLS_HANDSHAKE_SERVER_USAGE \
	"\ntls_handshake -s\n"             \
	"example: tls_handshake -s\n"

#define TLS_HANDSHAKE_CLIENT_USAGE    \
	"\ntls_handshake -c target_address\n" \
	"example: tls_handshake -c 127.0.0.1\n"

#define TLS_HANDSHAKE_USAGE        \
	"usage: tls_handshake mode\n" \
	"mode : client, server\n" 
```

Reply:
Thanks for your comment. I'll update it.